src/stactools/cli/__init__.py

```diff
@@ -35,15 +48,23 @@ def register_plugin(registry: "Registry") -> None:
     registry.register_subcommand(info.create_info_command)
     registry.register_subcommand(info.create_describe_command)
     registry.register_subcommand(layout.create_layout_command)
-    registry.register_subcommand(lint.create_lint_command)
     registry.register_subcommand(merge.create_merge_command)
     registry.register_subcommand(migrate.create_migrate_command)
     registry.register_subcommand(summary.create_summary_command)
-    registry.register_subcommand(validate.create_validate_command)
     registry.register_subcommand(version.create_version_command)
     registry.register_subcommand(update_extent.create_update_extent_command)
     registry.register_subcommand(update_geometry.create_update_geometry_command)
 
+    if HAS_STAC_VALIDATOR:
+        from stactools.cli.commands import validate
+
+        registry.register_subcommand(validate.create_validate_command)
```

Comment:
I guess my only question is what happens if a user tries to run `stac validate` and they don't have the deps? Ideally that would result in a nice error message that tells them what to do. So maybe that would lend itself to always registering and then have the error raised within the actual call to do validation?

Reply:
Hm, that's a good point. I can make a follow-on that makes the UX a bit nicer.

Reply:
I see people writing `import_optional` helpers a lot. Ala https://github.com/stac-utils/xpystac/blob/main/xpystac/utils.py#L5-L10